custom/bihar/management/commands/bihar_cleanup_case_2.py

```diff
@@ -58,9 +58,9 @@ def handle(self, *args, **options):
                     if case.user_id == MOTECH_ID:
                         case.type = "task"
                         self.cases_to_save[case.case_id] = case
-                        print(u"Case with name " + case.name + u" updated to type " + case.type)
+                        print("Case with name " + case.name.encode('ascii', 'xmlcharrefreplace') + " updated to type " + case.type)
                     else:
-                        print(u"Type not updated for case with name " + case.name)
+                        print("Type not updated for case with name " + case.name.encode('ascii', 'xmlcharrefreplace'))
```

Comment:
Seems fine, though I would have thought if you used u"{} some thing".format(unicode_name) that would also work (maybe?). also, is this just for console output? why not just print IDs instead? are you going to do anything with the hindi names?